### DISCUSSION TITLE
seekingalpha

### DISCUSSION THREAD

**Discussion by gene-git** [ranaroussi/yfinance#1515] [Ideas]
Have you thought about doing something similar for seekingalpha website - don't know how easy it would be to leverage existing work? Unclear if data offers any advantage or if its simpler or more complex.  Some find their website a more pleasant experience and interesting analyses.

 e.g. 

           https://seekingalpha.com         
           https://seekingalpha.com/symbol/SPY/historical-price-quotes

etc.

**Comment by ValueRaider** [comment]
Is it already implemented in here https://github.com/search?q=seekingalpha&type=repositories ?

**Comment by gene-git** [comment]
Indeed  it is :)
